swagger-spec/users/addons_list.yaml

```diff
@@ -1 +1,155 @@
 # /users/{user_id}/addons/
+get:
+  summary: List all addons
+  description: >-
```

Comment:
I think it would be useful to add a "Permissions" section in this description that explains that this endpoint can only be viewed for yourself (i.e. you can't see another user's addons list).